blog/views.py

```diff
@@ -0,0 +1,40 @@
+from django.http import HttpRequest, HttpResponse
+from django.views import generic
+from django.views.generic.edit import FormMixin
+
+from blog.forms import CommentaryForm
+from blog.models import Post
+
+
+class PostListView(generic.ListView):
+    model = Post
+    paginate_by = 5
+    ordering = ["-created_time"]
+
+
+class PostDetailView(FormMixin, generic.DetailView):
+    model = Post
+    queryset = Post.objects.prefetch_related("commentaries")
```

Comment:
model and queryset are interchangeable, so you should have either model or queryset